### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application.

### DISCUSSION THREAD

**Discussion by ElmirSultan** [community/community#70225] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

hi, I have a problem when I push my codes into my GitHub page. I get an error "You are marked as spam, and therefore cannot authorize a third party application.". I want you to help me about this. And also I have 24 stars here. But I had never starred somebody's project like 24 times.

**Comment by cehfisher** [comment]
Hi @ElmirSultan - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @ElmirSultan ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by ErfanMomeniii** [comment]
Hi, I have the same problem. can anyone help me? I created a ticket but haven't received any response from the support team. It's essential for me to change my account to the normal state because I can't make any contributions and my account is hidden now.
@cehfisher @queenofcorgis 
Ticket: https://support.github.com/ticket/personal/0/2979214
Account: https://github.com/erfanmomeniii

**Comment by Farman-Khan2242** [comment]
Hi I have the same problem can somebody help me plzzz i can't deployed my repos help me
